cardano-launcher/test/LauncherSpec.hs

```diff
@@ -202,3 +207,58 @@ withSetEnvs path homePath action = bracket
         , "DAEDALUS_INSTALL_DIRECTORY"
         , "LAUNCHER_CONFIG"
         ]
+
+generateTLSCertSpec :: Spec
```

Comment:
Niiice, Hiroto.
Very nice.

:tada: 

You are using the `withSystemTempDirectory` function to test it, which is good in this case.
In the future, it would be slightly better (in some cases, this is good as it is) to actually provide a stub function where you generate the TLS - this would make the code more flexible.